data/json/effects.json

```diff
@@ -4475,7 +4475,7 @@
           { "value": "ARMOR_BASH", "add": 15 },
           { "value": "ARMOR_STAB", "add": 15 },
           { "value": "ARMOR_CUT", "add": 15 },
-          { "value": "ARMOR_HEAT", "add": 15 },
+          { "value": "ARMOR_HEAT", "add": -5 },
```

Comment:
first, enchantments use reverse logic - negative values subtract amount of damage you receive, positive adds
second:
`"//": "Enchantments don't actually work for monsters! If you find that shadow is suddenly very killable, they probably got hooked up!",`

Reply:
Do you think it's in scope to fix this todo, or not so much?

Reply:
> first, enchantments use reverse logic - negative values subtract amount of damage you receive

Call me crazy but I find that to be a very confusing implementation.

Reply:
it's confusing, but it's how it works deep in the code, enchantments are just "okay you see this result? i want it to be added/subtracted/multiplied/divided"
It works well for melee damage, it works well for hp, it doesn't works as well with armor or recoil modifier

Reply:
Very confusing, yes, agreed.

Definitely, absurdly not in scope to hook up enchantments.